pkg/systemtemplate/template.go

```diff
@@ -193,6 +147,11 @@ spec:
       containers:
         - name: cluster-register
           imagePullPolicy: IfNotPresent
+          resources:
```

Comment:
If ResourceRequirements are nil we would rather not render empty `resources:` field.